### DISCUSSION TITLE
Diaper changes on dashboard

### DISCUSSION THREAD

**Discussion by matthieu-kr** [babybuddy/babybuddy#510] [Ideas]
Really great product! Definitely appreciate what you've done here. 

Just one thought - Any reason the dashboard only shows wet/solid changes? It kind of misrepresents the total number of changes since the sum of the two categories is greater than the total. I think it might make more sense to include a "both" category so that the sum of the changes on the bar is the same as the total changes.

Current state - 6 total changes; graph shows 5 solid and 5 wet (sum of 10)
![Graph 1](https://user-images.githubusercontent.com/34589622/182466239-d9b30184-688f-4c6c-8cc5-c3b6257afd6b.jpg)

Proposed - 6 total changes; graph shows 4 both, 1 solid, 1 wet (sum of 6)
![Graph 2](https://user-images.githubusercontent.com/34589622/182467127-e855bc60-2b35-45c7-ab6e-5f42a8cfe257.jpg)

**Comment by cdubz** [comment]
Hm yes I think there is room for improvement here… it would be nice to be able to represent both in some meaningful way. One issue though is that it’s also possible for a change to marked as neither wet nor solid. The proposed change here may get really cramped in that situation. It already gets cramps in certain use cases anyway.

**Comment by matthieu-kr** [comment]
Ahhhh. Hadn’t considered the fourth case. Perhaps it has a place in reporting?

**Comment by TaddyJ** [comment]
Another option could be instead of having the labels. Just keep the colours if it gets too small.
and maybe next to the title or a footer have what the colours mean?

**Comment by cdubz** [comment]
Hm yeah a legend could work.
